### ISSUE TITLE
Does not compile on nightly

### ISSUE THREAD

**Issue by Keats** [sfackler/rust-postgres-macros#12]
I'm running on nightly (1.5.0-nightly (6108e8c3e 2015-09-28)) and I get the following:

```rust
/postgres_macros-0.1.3/src/lib.rs:113:17: 113:53 error: this function takes 2 parameters but 1 parameter was supplied [E0061]
/postgres_macros-0.1.3/src/lib.rs:113     let ident = Ident::new(token::intern("execute"));
```

I had a look at Ident::new and token::intern but they both seem to only take 1 parameter so I'm not sure what's going on.

**Comment by zsiciarz** [comment]
Looks like the single-argument constructor is now called [with_empty_ctxt](https://github.com/serde-rs/syntex/blob/8ba8a2a17a47ad0c86a0ca12b5490dd216ac2e73/syntex_syntax/src/ast.rs#L129).

**Comment by sfackler** [comment]
Merged @zsiciarz's PR - should be fixed now. Published as v0.1.4.
